src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -10,12 +10,13 @@
 using Stratis.Bitcoin.Signals;
 using Stratis.Bitcoin.Utilities;
 using System.Timers;
+using Stratis.Bitcoin.Interfaces;
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
     public class BlockStoreSignaled : SignalObserver<Block>
```

Comment:
Is this missing a summary?   Whilst not normally a fan of comments, I can't see from the name BlockStoreSignaled what this class is responsible for so perhaps a very short summary (or a better name) would be useful.

Reply:
Agreed, but it's outside of the scope of this PR.

Reply:
ok